src/main/java/Event.java

```diff
@@ -0,0 +1,32 @@
+import java.time.LocalDate;
+import java.time.format.TextStyle;
+import java.util.Locale;
+public class Event extends Task {
+    protected LocalDate from;
```

Comment:
Is it necessary your variables to have access modifier: protected as opposed to private. As you have not implemented package, having it as protected doesn't provide additional utility?